### ISSUE TITLE
.append() is not necessarily jQuery

### ISSUE THREAD

**Issue by ptrkcsk** [Automattic/VIP-Coding-Standards#650]
## Bug Description

**Expected behavior**: No issue with using [`URLSearchParams.append()`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/append)

**Actual behavior**:  The following warning:

```
Any HTML passed to `append` gets executed. Make sure it's properly escaped.
```

It seems that VIPCS assumes `.append()` means [`jQuery.append()`](https://api.jquery.com/append/)

## Minimal Code Snippet

```js
var params = new URLSearchParams();
params.append('name', 'value');
```

## Error Code

WordPressVIPMinimum.JS.HTMLExecutingFunctions.append

## Environment

| Question                | Answer
| ------------------------| -------
| PHP version             | 7.4.16
| PHP_CodeSniffer version | 3.5.8
| VIPCS version           | 2.0.0

## Tested Against `master` branch?

- [ ] I have verified the issue still exists in the `master` branch of VIPCS.
- [ ] I have verified the issue still exists in the `develop` branch of VIPCS.

**Comment by GaryJones** [comment]
As the JavaScript sniffs have been deprecated for VIPCS 3.1.0, I'm closing this out.
